packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -52,8 +60,9 @@ const enableKernelGC = true;
 // allowed to vary between instances in a consensus machine. Everything else
 // is required to be deterministic.
 //
-// The schema is:
+// The current ("v1") schema is:
```

Comment:
```suggestion
// The schema is indicated by the value of the "version" key, which was added
// for version 1 (i.e., version 0 had no such key).
// Version 1 also added keys "kernel.defaultReapDirtThreshold" and
// "v$NN.reapDirt", dropped keys "kernel.defaultReapInterval",
// "v$NN.reapInterval",  and "v$NN.reapCountdown", and updated the schema for
// values of "v$NN.options" keys.

// As of version 1, known keys and values are:
```

Reply:
We've got the 0->1 delta information at the end of the table too.. which side do you think is better? We probably don't need to list it twice.

Reply:
ok I'm going to stick with the deltas at the end, I like the format better.